dev-haskell/polyparse/polyparse-1.13.ebuild

```diff
@@ -29,5 +29,6 @@ src_prepare() {
 	default
 
 	cabal_chdeps \
-		'base >= 4.3.1.0 && < 4.14' 'base >= 4.3.1.0'
+		'base >= 4.3.1.0 && < 4.14' 'base >= 4.3.1.0' \
+		'bytestring >= 0.9.1.0 && < 0.11' 'bytestring >= 0.9.1.0'
```

Comment:
This package has these changes already in the [upstream Hackage revision](https://hackage.haskell.org/package/polyparse-1.13/revisions/), so I would suggest using that instead [as I suggested](https://github.com/gentoo-haskell/gentoo-haskell/pull/1341#discussion_r1001901891) in your other PR.